File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/comment/direct/Java/Python/s684030429_commented.py
class Main:
    def __init__(self):
        import sys
        self.tokens = iter(sys.stdin.read().split())
        
    def ni(self):
        return int(next(self.tokens))
    
    def solve(self):
        a = self.ni()
        b = self.ni()
        print(max(0, a - 2 * b))
        
if __name__ == "__main__":
    Main().solve() # End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/comment/direct/Java/Go/s684030429_commented.go
package main

import (
	"fmt"
)

func main() {
	solve()
}

func solve() {
	var a, b int
	fmt.Scan(&a, &b)
	res := a - 2*b
	if res < 0 {
		fmt.Println(0)
	} else {
		fmt.Println(res)
	}
}

// End of Code
